### PR TITLE
Td/lu 5603

### PR DESCRIPTION
**Pull request by RyanAldy**

### CODE REVIEW

File: bastionPolicy.tf [hunk 1]
```diff
@@ -1,5 +1,9 @@
-data "aws_iam_group" "AllowTakeonBastionAccess" {
-    group_name = "AllowTakeonBastionAccess"
+# data "aws_iam_group" "AllowTakeonBastionAccess" {
+#     group_name = "AllowTakeonBastionAccess"
```
Comment (mattprice-28) [inline bastionPolicy.tf]: Can this commented code be removed

File: securityGroupNode.tf [hunk 2]
```diff
@@ -58,8 +58,10 @@ resource "aws_security_group" "node-securitygroup" {
     tags = {
         Name = "takeon-nodes"
         App = "takeon"
-        "aws:cloudformation:stack-id" = "arn:aws:cloudformation:eu-west-2:${var.accountID}:stack/node-stack/80d89740-f1ba-11e9-9980-02ea235e602c"
-        "aws:cloudformation:stack-name" = "node-stack"
+        "aws:cloudformation:stack-id" = "${aws_cloudformation_stack.node-stack.id}"
+        #"aws:cloudformation:stack-id" = "arn:aws:cloudformation:eu-west-2:${var.accountID}:stack/takeon-dev-sandbox-node-stack/0c614590-5dfa-11ea-b14c-021679f87d94"
```
Comment (mattprice-28) [inline securityGroupNode.tf]: same here, remove commented code

**Review by mattprice-28 — COMMENTED** [review]
Couple of lines of code commented out that could be removed

File: bastionPolicy.tf (hunk 1 above)
Comment (RyanAldy) [reply bastionPolicy.tf]: Yep, no idea why I still had it left in! I shall update this now

**Review by sivagudur — APPROVED** [review]

**Review by mattprice-28 — APPROVED** [review]

**Pull request merged**